src/components/document-info-cell.yml

```diff
@@ -0,0 +1,6 @@
+createdOn:
+  en: Created
+  de: Erstellt
+updatedOn:
+  en: Last update
+  de: Letzte Änderung
```

Comment:
All their brothers and sisters mostly live in `common.yml` (but maybe those are used more often than once), I wouldn't mind having those two there, too, but just a "gut feeling", no urge.

Reply:
These two are more custom translations, they diverge from the common (they are shorter versions basically).